src/controller/index.js

```diff
@@ -33,7 +33,8 @@ class Controller extends ENIP {
             },
             subs: new TagGroup(compare),
             scanning: false,
-            scan_rate: 200 //ms
+            scan_rate: 200, //ms,
+            connectedMessaging: connectedMessaging,
```

Comment:
ES6 sugar, change `connectedMessaging: connectedMessaging` to `connectedMessaging`

Reply:
```suggestion
            connectedMessaging,
```